gitlint/linters.py

```diff
@@ -19,6 +19,7 @@
 import os.path
 import string
 import subprocess
+import re
```

Comment:
Could you add this line just after os.path

Reply:
Done